llvm/utils/lit/tests/Inputs/shtest-cat/cat-e.txt

```diff
@@ -0,0 +1,33 @@
+## Tests to check cat -e output for ASCII characters and newlines
+
+# RUN: cat -e %S/txt-files/allchars | FileCheck %s --check-prefix=ALLCHARS
+# RUN: cat -e %S/txt-files/newline | FileCheck %s --check-prefix=NEWLINE
+
```

Comment:
Can you add a comment here noting that allchars is for testing metacharacter output w/ `-e`?
```suggestion
## Test output of -e for all ASCII characters including  ^ and M- notation

```

Reply:
done